js/deno.ts

```diff
@@ -81,6 +81,7 @@ export {
 export { inspect } from "./console";
 export { build, platform, OperatingSystem, Arch } from "./build";
 export { version } from "./version";
+export { getRandomValues, getRandomValuesSync } from "./get_random_values";
```

Comment:
Have a look in js/globals.ts 

you can export symbols globally there.

Reply:
Done